controllers/authController.js

```diff
@@ -4,6 +4,7 @@ const { createTkn } = require('../utils')
 exports.login = async (req, res) => {
   try {
     const user = await req.db.User.findOne({ ...req.body })
+    req.log.info(user)
```

Comment:
I would remove this comment.

Reply:
Ups, forgot it there. Will do by tomorrow morning 😅

Reply:
resolved this last night